src/lib/include/libsed.h

```diff
@@ -70,6 +70,22 @@ int sed_lock_unlock(struct sed_device *dev, const struct sed_key *key, enum SED_
  */
 int sed_reverttper(struct sed_device *dev, const struct sed_key *key, bool psid);
 
+/**
+ * Revert Locking SP to its Original Factory State.
+ *
+ * @param dev			the device to operate on
+ * @param key			the SID password
```

Comment:
It should be Admin1 password

Reply:
Yep, SID is not correct here as this is used for TPer Owner, initially it is the same as activate SP is copying SID to Admin1.